asset/css/markbind.css

```diff
@@ -84,6 +84,14 @@ code.hljs.inline {
   * This section covers the common styles used within Header, Footer and Side Navigation bars
   */
 
+/* Header */
+
+header.header-fixed {
+    position: fixed;
+    width: 100%;
+    z-index: 9999;
```

Comment:
Is there a reason why `9999` was chosen? We seem to be using `z-index: 1000` for the navbar. Should we just stick to that instead?

Reply:
It's just an arbitrary large integer, shouldn't matter much.